### DISCUSSION TITLE
Would be cool if we could change the start button icon and the image behind the taskbar icons.

### DISCUSSION THREAD

**Discussion by WoolenDoolen** [valinet/ExplorerPatcher#522] [General]
I do like the windows 10 taskbar functionally, and it looks fine too. But I think it would be cool if we could customize it more. Start 11 has that and i liked it but its paid and only has a 30 day trial. Just a recommendation.

**Comment by BraINstinct0** [comment]
#472 #436 #142 #383 #369 #339 
Main one would be #436

**Reply by valinet** [reply]
Yeah, I don’t get this habit of duplicating and continuously opening new issues and feature requests about this either. It’s not going to write itself or speed it up in any way, so I don’t get the point, really, and I can’t do everything at once. There was a contribution regarding this some time ago, actually some nice pointers were given, we had a good conversation, the code was okay but not production ready so to say, there were still issues to iron out, and neither I, nor the OP had time to develop the work on that. When either of us will have the time and mood or whenever and whether anyone else feels like properly implementing and contributing it, it will be shipped and that’d be it. But until then, duplicate threads help in no way, I understood the point from the first few posts, and neither does voting or similar stuff like that help. Voting or “+1”s or whatever doesn’t make the function implement itself and the code write itself up, so yeah, what is helpful is actual contributions that can we can work on and polish together to the standard needed for shipping in a release.

**Reply by valinet** [reply]
Also, people should really keep in mind that the aim of this project is not to get Start11 or StartAllBack or whatever else out of business or duplicate ALL their features etc. What I work on I decide based on what I have a need or an interest personally to work on at a specific moment, I don’t have a roadmap for this, and I don’t envision one either, neither personal, nor community driven. I don’t work on feature requests necessarily, I just work on what I feel like working… those can sometimes overlap, but not necessarily so. Also, that being said, I am more than open to good written contributions, it’s just that, at the scale this project is at, I don’t fancy its development being guided by anything else than what I feel like working on at whatever point in time. This was born out of my initial desire to learn about this stuff, share back with the world so as to offer a strong educational base, enable others to learn stuff and to enable further or future developments and build a sensible *developers* community around this, we are on GitHub after all, with the bonuses of offering a better experience on Windows 11 and providing a libre implementation of this kind of utility to a somewhat technical audience. Some goals were met reasonably well, some failed miserably and are actually quite a disappointment after the months I spent working on this. I feel like some people, and not speaking about this post, but in general, kind of miss these points.
